src/gatsby/src/components/park/parkDates.js

```diff
@@ -8,6 +8,88 @@ import { faChevronUp, faChevronDown } from "@fortawesome/free-solid-svg-icons"
 import HtmlContent from "./htmlContent"
 import SubArea from "./subArea"
 
+export const ReservationButtons = ({ campingTypeCode, parkOperation }) => {
+  const reservationUrlRules = {
+    "backcountry-camping": {
+      buttons: [
+        {
+          label: "Get permit",
+          fieldName: "backcountryPermitUrl"
+        },
+        {
+          label: "Make a reservation",
+          fieldName: "backcountryReservationUrl"
+        },
+        {
+          label: "Reserve canoe circuit",
+          fieldName: "canoeCircuitReservationUrl"
+        },
+        {
+          label: "Book groupsite",
+          fieldName: "backcountryGroupReservationUrl"
+        },
+        {
+          label: "Book shelter",
```

Comment:
I think backcountry shelter goes under the facility-type shelters
see https://alpha-dev.bcparks.ca/garibaldi-park/#facilities

Reply:
On PROD, "Elfin Lakes shelter" belongs to the "Cabins and huts". Could shelters be either facility or camping depending on its park?